src/Mealz/MealBundle/Resources/views/Meal/partials/meal.html.twig

```diff
@@ -10,19 +12,20 @@
     {% if bookedDishCombination|length > 0 %}
         {% set bookedDishTitles = bookedDishCombination.toArray|map(d => d.title) %}
         {% set combinedDishIDs = bookedDishCombination.toArray|map(d => d.slug)|join(',') %}
+        {% set canEditCombinedDish = available and bookedDishCombination.containsDishVariation %}
     {% endif %}
 {% endif %}
 
-<div class="meal-row combined-meal"
-    {%- if available -%}
+<div class="meal-row{{ dish.isCombinedDish ? ' combined-meal' }}"
+    {%- if meal.open -%}
+        data-id="{{ participant ? participant.id }}"
         data-combined="{{ combinedDish }}"
         data-slug="{{ dish.slug }}"
```

Comment:
Still need the slugs for the update method (meal open or locked or expired).

Reply:
Slugs and other data attributes are available as long as the meal is open (not expired). May be I didn't get your point here.

Reply:
My participant count updates include expired meals. I need to change this logic but for the current implementation, this breaks the updates.

Reply:
Check your js console: 
```
Warning: Values for count status update undefined. No values on 2022-01-24 for meal 12 and dish undefined
participation-count-update-handler.ts:50 Warning: Values for count status update undefined. No values on 2022-01-24 for meal 13 and dish undefined
participation-count-update-handler.ts:50 Warning: Values for count status update undefined. No values on 2022-01-24 for meal 63 and dish undefined
participation-count-update-handler.ts:50 Warning: Values for count status update undefined. No values on 2022-01-25 for meal 14 and dish undefined
participation-count-update-handler.ts:50 Warning: Values for count status update undefined. No values on 2022-01-25 for meal 15 and dish undefined
participation-count-update-handler.ts:50 Warning: Values for count status update undefined. No values on 2022-01-25 for meal 64 and dish undefined
participation-count-update-handler.ts:50 Warning: Values for count status update undefined. No values on 2022-01-26 for meal 16 and dish undefined
...
```

Reply:
We could optimize this in the future (not include updates for expired meals) but please for now move slugs outside of "meal.open".

Reply:
Done